standard.md

```diff
@@ -6,18 +6,17 @@
-**9a) Privacy & Freedom of Expression** |	All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on privacy and freedom of expression where governments are believed to be using the project’s product or services for illegitimate or political purposes.
-**9b) Data Privacy & Security** |	Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from it’s collection, storage and distribution.
-**9c) Inappropriate & Illegal Content** |	Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials and mechanisms for detecting and moderating inappropriate/illegal content.
-**9d) Protection from harassment** |	If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users.
+**6. Mechanism for extracting data** |	If this project has non personally identifiable information there must be a mechanism for extracting or importing non personally identifiable information (PII) data from the system in a non-proprietary format.
+**7. Adherence to privacy and applicable laws** |	The project must state that it complies with relevant privacy laws, and all applicable international and domestic laws. Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from its collection, storage and distribution.
+**8. Adherence to standards & best practices** |	Projects must demonstrate adherence to standards, best practices and/or principles. i.e. the principles for digital development.
+**9. Freedom of expression** |	All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on freedom of expression where governments are believed to be using the project’s product or services for illegitimate or political purposes.
+**10. Inappropriate & illegal content** |	Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials and mechanisms for detecting and moderating inappropriate/illegal content.
+**11. Protection from harassment** |	If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users.
```

Comment:
Yes, it's interesting to note how as we have started using the standard to screen DPGs, they refer to how they manage their communities and related policies to address this point (see [here](https://github.com/unicef/publicgoods-candidates/blob/5a333d27f516c26483ba503cd3a78499f7f7e1a0/screening/apache-fineract.json#L53) and [here](https://github.com/unicef/publicgoods-candidates/blob/4d3ade3872ae54081503e619a71b5d767702e8d0/screening/modular-open-source-identity-plaform.json#L117) for example). For the time being, we rely on this information as self-reported; but would be interesting to incorporate metrics here, maybe not as much as a full validation, but as a proxy or indicator for what they are claiming. Since this is not actionable currently, I'm marking this conversation as resolved.